assembl/static2/css/components/emoticons.scss

```diff
@@ -0,0 +1,19 @@
+.sentiment {
```

Comment:
Honestly, I didn't think about a reusable css for both versions. So it's separated and I think it's simplest.